pkg/check/pingpong/metrics.go

```diff
@@ -1,24 +1,40 @@
 package pingpong
 
-import "github.com/prometheus/client_golang/prometheus"
-
-var (
-	rttGauge = prometheus.NewGaugeVec(
-		prometheus.GaugeOpts{
-			Namespace: "beekeeper",
-			Subsystem: "check_pingpong",
-			Name:      "rtt_duration_seconds",
-			Help:      "Ping round-trip time duration Gauge",
-		},
-		[]string{"node", "peer"},
-	)
-	rttHistogram = prometheus.NewHistogram(
-		prometheus.HistogramOpts{
-			Namespace: "beekeeper",
-			Subsystem: "check_pingpong",
-			Name:      "rtt_seconds",
-			Help:      "Ping round-trip time duration Histogram",
-			Buckets:   prometheus.LinearBuckets(0, 0.003, 10),
-		},
-	)
+import (
+	m "github.com/ethersphere/bee/pkg/metrics"
+	mm "github.com/ethersphere/beekeeper/pkg/metrics"
+	"github.com/prometheus/client_golang/prometheus"
 )
+
+type metrics struct {
+	RttGauge     *prometheus.GaugeVec
+	RttHistogram prometheus.Histogram
+}
+
+func newMetrics() metrics {
+	subsystem := "check_pingpong"
+	return metrics{
+		RttGauge: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: mm.Namespace,
+				Subsystem: subsystem,
+				Name:      "rtt_duration_seconds",
+				Help:      "Ping round-trip time duration Gauge",
```

Comment:
To stay consistent add the missing period (`.`) at the end of the sentence, also below.